public/js/index.js

```diff
@@ -128,250 +133,196 @@ function handleViewClick(lesson) {
 	overlay.classList.add('dark');
 }
 
-function lessonHelper({varName, eventListener, classList, attribute, textContent, innerHTML, id}) {
```

Comment:
I don't think we want to remove this helper function. Not sure if this was intentional or not.

Reply:
Oh god, I may have deleted it accidentally. I will re-add it.

Reply:
I re-added it with the following commit: 74fb323c6b210cca11ba958f1ea9206b14279fb1